deploy/crds/apps_v1alpha1_apimanager_crd.yaml

```diff
@@ -60,14 +65,22 @@ spec:
             system:
               properties:
                 database:
+                  description: TODO should union fields be optional?
                   properties:
                     mysql:
+                      description: Union type. Only one of the fields can be set
                       properties:
                         image:
                           type: string
                       type: object
                   type: object
                 fileStorage:
+                  description: TODO should this field be optional? We have different
```

Comment:
Same about some comments appearing that did not appear before. This was really an internal comment